### ISSUE TITLE
Some input fields have placeholder text that overlaps and persists

### ISSUE THREAD

**Issue by mikeLspohn** [brave/browser-laptop#403]
On the homepage of The Pirate Bay the input field has two placeholder texts that over lap and one of them persists, even after you start entering text in the input field. 

<img width="415" alt="screenshot 2016-01-28 21 01 46" src="https://cloud.githubusercontent.com/assets/6027057/12666054/8222310c-c602-11e5-8f60-9903f2f785dc.png">

I checked on Chrome and Firefox and that's not happening there, so it's not the website. I haven't ran into this on any other site yet. I'm on OSX Yosemite, node version 5.5.0.

**Comment by inian** [comment]
Am able to reproduce the bug with the following HTML document
```
   <html>
   <input type="search" placeholder="hello"></input>
   </html>
```

**Comment by psimyn** [comment]
@mikeLspohn can you still reproduce this on latest version?

**Comment by bbondy** [comment]
Still reproducible:
data:text/html,<html>%20<input%20type="search"%20placeholder="hello"></input></html>

**Comment by mikeLspohn** [comment]
Yep, still reproducible for me as well using the 0.7 OSX.

**Comment by Liunkae** [comment]
Still reproducible with Brave 0.12.3 on macOS 10.12.

**Comment by spongessuck** [comment]
Reproducible with Brave 0.12.5 on Mac OS X 10.11.

**Comment by Liunkae** [comment]
Still reproducible with Brave 0.13.0 on macOS 10.12.x.

**Comment by Liunkae** [comment]
Still reproducible with Brave 0.14.0 (c23efa3, Muon 2.57.7, libchromiumcontent 57.0.2987.110) on macOS 10.12.x

**Comment by Liunkae** [comment]
Still reproducible with Brave 0.15.0 (71fb730, Muon 2.58.7, libchromiumcontent 58.0.3029.81) on macOS 10.12.x.

**Comment by Liunkae** [comment]
Still reproducible.

![image](https://user-images.githubusercontent.com/19424103/27658733-913bba90-5c16-11e7-80ca-f253c4253207.png)

Brave: 0.17.12 
rev: 0095c359d27c5387a07152fdec161dfc54c5fef4 
Muon: 4.1.6 
libchromiumcontent: 59.0.3071.109 
V8: 5.9.211.38 
Node.js: 7.9.0 
Update Channel: dev 
os.platform: darwin 
os.release: 16.6.0 
os.arch: x64

**Comment by Liunkae** [comment]
This is **fixed** as of Brave 0.19.70 on macOS 10.13.

**Comment by luixxiul** [comment]
thanks for confirming that :-)
